Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN curl -sSL https://install.python-poetry.org | python3 -
 ENV PATH=$PATH:/root/.local/bin/
 WORKDIR /app
```

Comment:
I think you need to change this to **WORKDIR todo_app/app**

I think this is causing the docker container to not run properly.